mini-wallet.md

```diff
@@ -52,86 +50,106 @@ Options:
   --help                          Show this message and exit.
 ```
 
+View the DiemMiniWallet API specification (OpenAPI Specification 3.0.3) by opening http://localhost:8888. These specs include simple examples to try out the MiniWallet API. A hosted version of the API specification document is also available [here](https://diem.github.io/client-sdk-python/mini-wallet-api-spec.html).
+
+
+## Use the Diem MiniWallet Test Suite
+
+The Diem MiniWallet Test Suite is a set of tests built on top of the Diem MiniWallet API and is used to validate a wallet app’s integration with the Diem Payment Network (DPN). 
+
+Use the MiniWallet Test Suite to automate testing and checking if your wallet application meets our requirements. 
+
+1. Hit the target server we started by `dmw start-server`:
+	`dmw test --target http://localhost:8888` 
+
+	You should see the following test report:
+
+	```
+	> dmw test
+	Diem JSON-RPC URL: http://testnet.diem.com/v1
+	Diem Testnet Faucet URL: http://testnet.diem.com/mint
+	======================================== test session starts 						===============================================
+	…...
+	collected 61 items
+	
+	src/diem/testing/suites/test_miniwallet_api.py 										......................ss.                              [ 40%]
```

Comment:
should not have new line, or you added too many empty space.

Reply:
removed empty space